### ISSUE TITLE
Missing tslib dependency

### ISSUE THREAD

**Issue by kazkansouh** [JonathanWilbur/asn1-ts#19]
Hello, thanks for making a nice project!

I noticed that after importing this project and compiling my (TypeScript) code into a common js module it was failing to run due to a missing dependency `tslib`.

```
Uncaught Error: Cannot find module 'tslib'
Require stack:
- /dev/shm/t/node_modules/asn1-ts/dist/node/index.js
```

I believe the issue is related to:

https://github.com/JonathanWilbur/asn1-ts/blob/951e31b76e6a6adbbd997f73a7aebba0c1c1fd9b/tsconfig.json#L19

From the TypeScript [documenation](https://www.typescriptlang.org/tsconfig#importHelpers):

> If the importHelpers flag is on, these helper functions are instead imported from the [tslib](https://www.npmjs.com/package/tslib) module. You will need to ensure that the tslib module is able to be imported at runtime. This only affects modules; global script files will not attempt to import modules.

I believe you need to do a simple `npm i tslib`?

**Comment by JonathanWilbur** [comment]
You're right! My apologies. Fixing now.

**Comment by JonathanWilbur** [comment]
Fixed with version `7.0.17`.
